src/main/java/com/trailfinder/dto/EventDTO.java

```diff
@@ -22,6 +21,38 @@
 	private ProfileDTO eventCreator;
 	private List<ProfileDTO> attendees;
 	
+	/**
+	 * Create and initialize a new instance of the EventDTO
+	 * 
+	 * @param eventId
+	 * @param distance
+	 * @param eventStart
+	 * @param eventEnd
+	 * @param latitude
+	 * @param longitude
+	 * @param eventCreator
+	 * @param attendees
+	 */
+	public EventDTO(int eventId, double distance, LocalDateTime eventStart, LocalDateTime eventEnd, double latitude, 
+			double longitude, ProfileDTO eventCreator, List<ProfileDTO> attendees) 
+	{
+		this.eventId = eventId;
+		this.distance = distance;
+		this.eventStart = eventStart;
+		this.eventEnd = eventEnd;
+		this.Latitude = latitude;
+		this.Longitude = longitude;
+		this.eventCreator = eventCreator;
+		this.attendees = attendees;
+	}
+	
+	/**
+	 * Create an instance of EventDTO
+	 */
+	public EventDTO() {
```

Comment:
Created default constructor for when you don't want to initialize fields during instantiation.

Reply:
These days, many classes only have a default constructor.  Unless:
1) There are attributes that are required for the object to be constructed... like a database connection.
2) You are using constructor dependency injection, in Spring, or 
3) You're using Kotlin, where attributes tend to be initialized as constructor parameters.